src/lib/styles/mixins/_toolbar.scss

```diff
@@ -7,23 +7,26 @@
   display: flex;
   gap: var(--padding-2x);
 
-  & > :global(button),
-  & > :global(.tooltip-wrapper) {
-    // use 50% to avoid non even button sizes (a single button would be 100%)
-    flex: 1 1 50%;
+  :global(button) {
+    // all buttons inside the toolbar should be full width on mobile and be aligned by flex parameters.
+    width: 100%;
   }
 
-  & > :global(.tooltip-wrapper) {
-    :global(*) {
-      width: 100%;
-    }
+  & > :global(*) {
+    // 50% to make sure the button is not smaller in the tooltip, because of the wrong width calculation.
+    flex: 1 1 50%;
```

Comment:
Does `50%` assume there are 2 buttons? What if there are 1 or 3?

Reply:
It looks best w/ 1 or 2 buttons, but also not so bad w/ 3+ because of `flex: 1 1` part:

<img width="490" alt="image" src="https://github.com/dfinity/gix-components/assets/98811342/67b85cec-8654-489c-a3a1-9cbc286d02bb">

<img width="433" alt="image" src="https://github.com/dfinity/gix-components/assets/98811342/ec4e154c-d162-4cb6-85e2-4002a5f3080c">

<img width="525" alt="image" src="https://github.com/dfinity/gix-components/assets/98811342/1827d2a5-7861-48ab-9b08-f3d5545e53a8">

Reply:
Then I'm not sure why it's `50%` and not something else. Would be good to know but not important for this PR.